docs/development/virtualizing_tails.rst

```diff
@@ -64,7 +64,7 @@ Next you will install Tails onto the Virtual Hard Disk Image. Start the VM, boot
 to Tails, and enter an administration password and start Tails.
 
 .. note:: For all the instructions that follow, you will need to configure an
-          administrator password each time you boot Tails.
```

Comment:
replace **administrator** with **administration** here because this is how it is called in Tails